tests/src/test/scala/quasiquotes/ErrorSuite.scala

```diff
@@ -48,7 +64,37 @@ class ErrorSuite extends FunSuite {
     """.trim.stripMargin)
   }
 
-  test("q\"...$xs\"") {
+  test("q\"foo($xs)\" when xs has incompatible type") {
+    assert(typecheckError("""
+      import scala.meta._
+      import scala.meta.dialects.Scala211
+      val xs = List(q"x")
+      q"foo($xs)"
+    """) contains """
+                               |<macro>:5: type mismatch when unquoting;
+                               | found   : List[scala.meta.internal.ast.Term.Name]
+                               | required: scala.meta.Term.Arg
+                               |      q"foo($xs)"
+                               |            ^
```

Comment:
Wdyt about adjusting indentation of this example to match the pre-existing style in the tests in this file?